go.mod

```diff
@@ -0,0 +1,27 @@
+module multiversx/mvx-keys-monitor
+
+go 1.20
+
+require (
+	github.com/BurntSushi/toml v0.3.1
```

Comment:
Seems to be a different `toml` library than the one used by `mx-chain-go`. It seems to be fine (many stars, many forks).

Reply:
autocompleted pe goland. Will switch to what we are using. Thanks for the heads-up. :+1:

Reply:
removed